python/metashape_workflow_functions.py

```diff
@@ -1255,3 +1266,21 @@ def finish_run(self):
             file.write("### END CONFIGURATION ###\n")
 
         return True
+
+    def get_paths_written(self):
```

Comment:
nitpick: it could be helpful to make the name of the accessor function and the member variable the same (`written_paths` vs. `paths_written`)